terraform/modules/aws_cargohold-asset-storage/README.md

```diff
@@ -0,0 +1,20 @@
+Terraform module: Cargohold Asset Storage
+=========================================
+
+This module creates an Object Storage on AWS for cargohold to store encrypted assets.
+
+
+#### How to use the module
+
+```hcl-terraform
+module "gundeck-push-notification" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws_cargohold-asset-storage?ref=develop"
```

Comment:
module name doesn't match the source (name is about gundeck, source abour cargohold). Rename?

Also, could we agree on all-underscores? It's slightly strange to have an underscore after aws but not between the other words. I'm not very opinionated about `_` or `-`, just I'd like to use them consistently, and not a mixture.

Reply:
Fair point. Consistency is a good thing. I now ended up using dashes in folders and underscores for any TF code. Would this be feasible?